packages/world-module-erc20/src/Constants.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+import { RESOURCE_TABLE } from "@latticexyz/store/src/storeResourceTypes.sol";
+import { ResourceId, WorldResourceIdLib } from "@latticexyz/world/src/WorldResourceId.sol";
+
+library ModuleConstants {
+  bytes14 constant NAMESPACE = "erc20-module";
+  bytes16 constant REGISTRY_TABLE_NAME = "ERC20_REGISTRY";
+
+  function namespaceId() internal pure returns (ResourceId) {
+    return WorldResourceIdLib.encodeNamespace(NAMESPACE);
+  }
+
+  function registryTableId() internal pure returns (ResourceId) {
+    return WorldResourceIdLib.encode(RESOURCE_TABLE, NAMESPACE, REGISTRY_TABLE_NAME);
+  }
```

Comment:
so these are only used in the module installation, the reason I added them as functions using WorldResourceIdLib instead of manual encoding is so that if at any point the `encode` implementation changes then it would break those other constants, but I can move if that's ok.